### PR TITLE
chore: covector init

### PR DESCRIPTION
**Pull request by jbolda**
🎉 

Note: There is an added option that you can specify `housekeeping` and it will effectively no-op on it. This way we can always require a change file.

## TODO
- [ ] enter the `crate_token`

### CODE REVIEW

**Comment by nothingismagick** [comment]
@jbolda - we have it already at org level.
image

However, @wusyong will have to add the tauri-bot user to the crate or we won't be able to push.

File: .github/workflows/covector-version-or-publish.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      change: ${{ steps.covector.outputs.change }}
+      commandRan: ${{ steps.covector.outputs.commandRan }}
+      successfulPublish: ${{ steps.covector.outputs.successfulPublish }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: cargo login
+        run: cargo login ${{ secrets.crate_token }}
```
Comment (jbolda) [inline .github/workflows/covector-version-or-publish.yml:22]: @nothingismagick I assume that means we need to switch this?

```suggestion
        run: cargo login ${{ secrets.ORG_CRATE_TOKEN }}
```

**Comment by wusyong** [comment]
I've added tauri bot to the crate owner. This means we can merge it right?

**Review by wusyong — APPROVED** [review]

**Comment by jbolda** [comment]
> I've added tauri bot to the crate owner. This means we can merge it right?

Aye, I think we are ready to see it fly.

**Pull request merged**